pom.xml

```diff
@@ -31,11 +31,11 @@
 
     <properties>
         <connect-runtime-version>2.0.0</connect-runtime-version>
-        <confluent.version>5.4.0</confluent.version>
+        <confluent.version>5.4.1</confluent.version>
         <confluent.avro.generator.version>0.3.0</confluent.avro.generator.version>
         <junit.version>4.12</junit.version>
         <avro.version>1.8.1</avro.version>
-        <licenses.version>5.4.0</licenses.version>
+        <licenses.version>5.4.1</licenses.version>
```

Comment:
```suggestion
```

Similarly to the `${confluent.version}` property, we can [remove this one too](https://github.com/confluentinc/common/blob/f668cc36070d6fdcbde9b7c9854e84d302c38346/pom.xml#L66).